### PR TITLE
Add --fio_blocksize option

### PR DESCRIPTION
**Pull request by noahl**
Lets users set the block size in the fio benchmark. This also required a
workaround for a bug in Pint; the fix has been submitted to upstream
Pint, but has not been merged yet.

### CODE REVIEW

**Comment by noahl** [comment]
@stfeng @yuyantingzero please take a look.

File: perfkitbenchmarker/linux_benchmarks/fio_benchmark.py [hunk 1]
```diff
@@ -124,10 +125,16 @@
                      'given number of minutes. Time will be rounded up to the '
                      'next multiple of %s minutes.' % MINUTES_PER_JOB,
                      lower_bound=0)
+flag_util.DEFINE_units('fio_blocksize', None,
+                       'The block size for fio operations. Default is given by '
+                       'the scenario when using --generate_scenarios and the '
```
Comment (yuyantingzero) [inline perfkitbenchmarker/linux_benchmarks/fio_benchmark.py]: The help message here is not clear to me. When using custom job file, this flag is not used.

File: perfkitbenchmarker/__init__.py [hunk 2]
```diff
@@ -39,3 +40,13 @@ def _UnPickleQuantity(inp):
 
 
 copy_reg.pickle(UNIT_REGISTRY.Quantity, _PickleQuantity)
+
+
+# The following monkey-patch has been submitted to upstream Pint as
+# pull request 357. TODO: once that PR is merged, get rid of this
```
Comment (yuyantingzero) [inline perfkitbenchmarker/__init__.py]: Move TODO to a separate line

File: tests/linux_benchmarks/fio_benchmark_test.py [hunk 3]
```diff
@@ -103,9 +104,17 @@ def testMultipleScenarios(self):
...
         expected_jobfile)
 
+  def testCustomBlocksize(self):
+    job_file = fio_benchmark.GenerateJobFileString(
+        self.filename,
+        ['sequential_read'],
+        [1], None, perfkitbenchmarker.UNIT_REGISTRY.megabyte * 2)
+
+    self.assertIn('blocksize=2000000B', job_file)
+
```
Comment (yuyantingzero) [inline tests/linux_benchmarks/fio_benchmark_test.py]: Also assert the global scenario variable not modified.

**Comment by yuyantingzero** [comment]
A few comments, LGTM otherwise

File: perfkitbenchmarker/linux_benchmarks/fio_benchmark.py (hunk 1 above)
Comment (noahl) [inline perfkitbenchmarker/linux_benchmarks/fio_benchmark.py]: I edited the help message to be more clear, thanks.

File: perfkitbenchmarker/__init__.py (hunk 2 above)
Comment (noahl) [inline perfkitbenchmarker/__init__.py]: Done.

File: tests/linux_benchmarks/fio_benchmark_test.py (hunk 3 above)
Comment (noahl) [inline tests/linux_benchmarks/fio_benchmark_test.py]: Done.

**Comment by noahl** [comment]
@yuyantingzero I just pushed a commit that I think addresses all of your comments. PTAL.

**Comment by yuyantingzero** [comment]
LGTM.

**Pull request merged**